userguide/tutorials/aws.adoc

```diff
@@ -38,39 +38,41 @@
 
 **Single AMI Deployments** come in 2 flavors:
 
-* `Single-Tier`: In order to quickly get started, we provide an AMI that can be launched with a **1-click** button. Both Kill Bill server, KAUI and the database run on **one node**, so while this is very convenient to get started, this is often a poor option for production deployments.
-* `Multi-Tier`: In this mode, there is also one single AMI for both Kill Bill server and KAUI, but the deployment will rely on an external database. Typically such deployments will also rely on at least 2 instances for running the Kill Bill stack to provide the right level of redundancy and zero downtime upgrade deployments.
+* `Single-Tier`: To help you to quickly get started, the single-tier deployment offers an AMI that provides everything you need and can be launched with a **one-click** button. In this option the Kill Bill server, KAUI and a MariaDB (MySQL) database run on one single node. This is very convenient and inexpensive to get started, but is not recommended for production deployments.
+* `Multi-Tier`: In this mode, there is also a single AMI for both Kill Bill server and KAUI, but the deployment will use more than one node (typically two). This option requires a bit more setup and depends on an external database. The use of multiple nodes is intended to provide redundancy and to eliminate downtime during upgrades.
 
-The Single AMI Deployments are a great way to easily get started through the `Single-Tier` option, but they typically require a bit more setup for production deployments. Please refer to the https://docs.killbill.io/latest/aws-single-ami.html[specifics of the Single AMI Deployments].
+The Single AMI Deployments are a great way to easily get started, but they are often not the best choices for production deployments. Please refer to https://docs.killbill.io/latest/aws-single-ami.html[Installation via Single AMI] for further information.
 
-The rest of this documentation focuses on configurations that apply to both types of deployments.
+**Cloud Formation Deployments** use AWS CloudFormation to provide a better integration with the rest of the AWS ecosystem. These deployments include separate AMIs for the Kill Bill Server and for Kaui, and rely on auto-scaling groups to scale both Kill Bill and KAUI instances independently. They make use of the Amazon Aurora RDS database, a version of MySQL that is also PostGres compatible. They also make use of CloudWatch for metrics.
```

Comment:
I had never seen postgres being written `PostGres`. Typically, we see `PostgreSQL`, `postgres`, `Postgres`. Maybe look around and confirm, and then based on what you find update the doc?